types/index.ts

```diff
@@ -12,7 +15,22 @@ export interface Contract {
   addresses: TaggedAddress[];
 }
 
+export interface ProcessedContract extends Contract {
+  network: string;
```

Comment:
Should this be a `Network` rather than a `string`?

Reply:
Nice catch. Thanks!

Reply:
`8289219524253097fac4247b40c3069ed06fb569`